roles/common/tasks/crd_tests.yml

```
@@ -0,0 +1,14 @@
+---
+- name: Verify CRD "{{ item }}"
+  ansible.builtin.shell:
+    cmd: |
+      oc get crd "{{ item }}"
+  changed_when: false
+  register: output
+
+- name: Verify CRD is found "{{ common_crd_test_id }}"
```

The test ID should be made optional OR this file should not be included unless it is defined (see comment below)

```suggestion
- name: Verify CRD is found {{ common_crd_test_id | default('') }}
```